enterprise_access/apps/api/tasks.py

```diff
@@ -127,6 +130,50 @@ def send_notification_email_for_request(
     )
 
 
+@shared_task(base=LoggedTaskWithRetry)
+def send_cancel_email_for_pending_assignment(cancelled_assignment):
+    """
+    Send email via braze for cancelling pending assignment
+
+    Args:
+        cancelled_assignment: (string) the subsidy request uuid
+    """
+    learner_content_assignment_action = LearnerContentAssignmentAction(
```

Comment:
OK, so remember from another comment that we should be passing an assignment uuid into this task - pretend it's called `cancelled_assignment_uuid`.  What you'll want to do is to first in this task is fetch the assignment record based on that uuid:
```python
cancelled_assignment = LearnerContentAssignment.objects.get(cancelled_assignment_uuid)
```

Reply:
Done!